gensim/summarization/mz_entropy.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
```

Comment:
Now coding style is better, but contains problems yet, please have a look to flake8 log - https://travis-ci.org/RaRe-Technologies/gensim/jobs/307965704#L517